ACM-8243: update hosted cluster destroy docs (#5670)
open-cluster-management/rhacm-docs: clusters/hosted_control_planes/destroying_hosted_bare_metal.adoc

. On the _Clusters_ page, select the cluster that you want to destroy.

. In the *Actions* menu, select *Destroy clusters* to remove the cluster.

[#hypershift-cluster-destroy-bm-cli]

> = Destroying a hosted cluster on bare metal using command line

Reviewer: ```suggestion
== Destroying a hosted cluster on bare metal by using the command line
```

To destroy a hosted cluster and its managed cluster resource, complete the following steps:

. Delete the managed cluster resource on {mce-short} by running the following command:

+
----
oc delete managedcluster <cluster_name>
----

+

> Replace `cluster_name` with the name of your cluster.

Reviewer: ```suggestion
where _`cluster_name`_ is the name of your cluster.
```
